x-pack/filebeat/input/awss3/config.go

```diff
@@ -127,6 +128,7 @@ type readerConfig struct {
 	LineTerminator           readfile.LineTerminator `config:"line_terminator"`
 	MaxBytes                 cfgtype.ByteSize        `config:"max_bytes"`
 	Parsers                  parser.Config           `config:",inline"`
+	Split                    *split.SplitConfig      `config:"split"`
```

Comment:
You have added the new parser named `split`. There is no need for adding `Split` configuration, too.

It should work like this:
```yaml
parsers:
  - split: ~
```

Reply:
That was before I started teh parser work, just haven't removed it yet.  focusing on one input to test with, the http endpoint input. Once its all working, I'll go through and remove the previous implementation of it being an independent config option.